src/app/static/view-question/view-question.component.html

```diff
@@ -2,14 +2,15 @@
   <app-question [question]="selectedQuestion"></app-question>
 </div>
 
-<br>
-<div class="msg-count"> total responses: {{totalMsgCount}}</div>
+<div class="msg-count"> 
+  total responses: {{selectedQuestion.responses.length}}
+</div>
 
 <div class="highlighted-response-component">
-  <app-highlighted-response *ngIf="highlightedResponse" [highlight] = "highlightedResponse"></app-highlighted-response>
+  <app-highlighted-response *ngIf="selectedQuestion.highlightedResponseId" [highnum] = "selectedQuestion.highlightedResponseId"></app-highlighted-response>
```

Comment:
This also applies elsewhere: I'd like a more descriptive name than 'highnum'.  I understand it as the id of the highlighted response, but only because I've been reading this code for awhile.